rmf_demos_panel/rmf_demos_panel/static/src/components/services.tsx

```diff
@@ -1,5 +1,3 @@
-import { showErrorMessage, showSuccessMessage } from './fixed-components/messages';
-
 const API_SERVER_ADD = "http://" + location.hostname + ":8080"
 const GUI_SERVER_ADD = "http://" + location.hostname + ":5000"
```

Comment:
Would like to sneak in this change:
```
const GUI_SERVER_ADD = "http://" + location.host
```

Reply:
oh okay!

and about the numerical input, sorry I think I missed it out! we are looking at only 0 and 1 or a range from 0 to 1? (0, 0.1, 0.2 ... 1.0)

Reply:
We can use [this](https://www.npmjs.com/package/react-numeric-input). The numerical input should be in integer, from 0-N.